atd-vzd/triggers/get_cr3_location_id.sql

```diff
@@ -0,0 +1,21 @@
+-- This function is invoked by a stored generated column called location_id in the atd_txdot_crashes table.
+-- It is responsible for finding the location id of a crash, setting the location_id to NULL if the crash occurred on a 
+-- level 5 roadway and thus is outside of COA jurisdiction.
+CREATE OR REPLACE FUNCTION get_cr3_location_id(location geometry(Geometry,4326), crash_id integer, rpt_road_part_id integer, rpt_hwy_num text) RETURNS text AS $$
+        BEGIN
+            -- If crash is level 5 return null because it is outside of COA jurisdiction
+            IF (is_crash_level_5(crash_id, rpt_road_part_id, rpt_hwy_num)) THEN
```

Comment:
instead of putting the logic for determining if a crash is level 5 within the code here, i decided to make a diff function that does that in case this logic needs to be reused again elsewhere in the future? lmk if yall think thats right. also lmk if you think there is a better descriptive name than `is_crash_level_5`

Reply:
im kinda lost on what terminology we are going with. "level 5", "mainlane", and "major road" are thrown around kinda interchangeably in our convos and it would be good to maybe pick one for describing these kinds of roads. i think we already discussed how mainlane isnt super accurate bc small neighborhood streets also have mainlanes, so either "level 5" or "major road" sounds good to me.